app/src/pages/inside/dashboardItemPage/widgetsGrid/emptyWidget/emptyWidget.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { Fragment, Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { injectIntl, defineMessages, intlShape } from 'react-intl';
+import { GhostButton } from 'components/buttons/ghostButton';
+import styles from './emptyWidget.scss';
+import AddDashboardIcon from '../../img/add-inline.svg';
+
+const cx = classNames.bind(styles);
+const messages = defineMessages({
+  addNewWidget: {
+    id: 'DashboardItemPage.addNewWidget',
+    defaultMessage: 'Add new widget',
+  },
+  dashboardEmptyText: {
+    id: 'DashboardItemPage.dashboardEmptyText',
+    defaultMessage: 'Add your first widget to analyse statistics',
+  },
+  notMyDashboardEmptyHeader: {
+    id: 'DashboardItemPage.notMyDashboardEmptyHeader',
+    defaultMessage: 'You have no widgets on this dashboard',
+  },
+});
+
+@injectIntl
+export class EmptyWidget extends Component {
```

Comment:
Looks like it's `EmptyDashboardView` or `EmptyWidgetGrid` or something like that because it is visible when there are no widgets on a dashboard